src/delegation/policy/selector/select.rs

```diff
@@ -46,9 +46,9 @@ impl<T> Select<T> {
 
 impl<T: Selectable> Select<T> {
     pub fn get(self, ctx: &Ipld) -> Result<T, SelectorError> {
```

Comment:
The `Try` behaviour in this method could probably be more elegant, but I really don't want to have to write more custom `Try` (AKA `.?`) parsing 😅